openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSConfigServiceImpl.java

```diff
@@ -92,5 +99,14 @@ private void updateConfigs() {
         String jsonText = gson.toJson(configs, Configs.class);
         ByteArrayResource resource = new ByteArrayResource(jsonText.getBytes());
         settingsFacade.saveRawConfig(OPEN_MRS_CONFIGS_FILE_NAME, resource);
+
+        Map<String, Object> parameters = new HashMap<>();
+        eventRelay.sendEventMessage(new MotechEvent(Constants.CONFIG_CHANGE_EVENT, parameters));
+    }
+
+
+    @Autowired
```

Comment:
Let's stick to one convention for autowiring beans. Either only setters, only fields or only constructor.